### ISSUE TITLE
Login failed

### ISSUE THREAD

**Issue by jhchrist** [einstein95/crunchy-xml-decoder#28]
The original version I found on some forum allowed me to login but I was getting some bs4 error so I deleted it. Found this git, downloaded the latest version, installed the prerequisites, and it is now saying login failed. I run _make_cookies.bat again and select no, but it says could not find cookies.txt. So I create a new text file, rename it, and select no again. It says no cookies created then it deletes the cookies.txt file.

Trying a new export with either the old cookies.txt from the first version I downloaded, or an empty cookies.txt file in the crunchy-xml-decoder-master dir results in an [error](http://i.imgur.com/MUhfnwY.png). Removing the extension does nothing. Switching the system locale back to English makes no difference. 

Is it broke or did I mess up installing something?

**Comment by MidgardSerpent** [comment]
"I run _make_cookies.bat again and select no, but it says could not find cookies.txt"
it looks like _make_cookies.bat has not been updated

run the _make_cookies.bat and press y to log back in

**Comment by jhchrist** [comment]
Sorry I wasn't clear, I ran the make cookies batch after downloading the new version and it said login failed after I input my login information. 

>it looks like _make_cookies.bat has not been updated

Not sure exactly what you mean here, but I didn't overwrite any files after downloading the new version, I deleted the old folder completely. I'm more of a C#/VB guy but I'll take a look at the source after work tomorrow morning see if I can't spot why this is going on.
